snippets/wpa-supplicant-debug/overlay-wpa-supplicant-debug.conf

```diff
@@ -9,13 +9,11 @@ CONFIG_SHELL_STACK_SIZE=4600
 CONFIG_NET_SHELL=y
 CONFIG_SHELL_GETOPT=y
 CONFIG_SHELL_CMDS_RESIZE=n
-CONFIG_NRF700X_UTIL=y
 CONFIG_NET_L2_WIFI_SHELL=y
 CONFIG_NET_STATISTICS=y
 CONFIG_NET_STATISTICS_WIFI=y
 CONFIG_NET_STATISTICS_USER_API=y
 CONFIG_SYS_HEAP_RUNTIME_STATS=y
 CONFIG_LOG=y
 CONFIG_PRINTK=y
-CONFIG_LOG_MODE_IMMEDIATE=y
```

Comment:
My comment https://github.com/nrfconnect/sdk-nrf/pull/16582#pullrequestreview-2201818511 doesn't ask to remove, it should be enabled, else we loose many logs and won't help in debugging, but add a note in the docs.